### DISCUSSION TITLE
Cannot find any model that supports emotion?

### DISCUSSION THREAD

**Discussion by chengstark** [coqui-ai/TTS#3006] [General Q&A]
Following the readme, there is no model name following coqui_studio/en/<studio_speaker_name>/coqui_studio format.
None of the TTS(cs_api_model="V1").list_models() supports emotion either, anyone knows what is the solution?

**Comment by erogol** [comment]
V1 models takes an additional argument for emotion. None of the open-source models support emotion currently.
